src/Microsoft.Azure.WebJobs.Host/Extensions/IConfigurationExtensions.cs

```diff
@@ -93,5 +93,37 @@ public static bool IsSettingEnabled(this IConfiguration configuration, string se
             
             return false;
         }
+
+        public static IConfigurationSection GetWebJobsConnectionStringSection(this IConfiguration configuration, string connectionStringName)
```

Comment:
Does this new public method have to live in this assembly, or will it work if it lives in the Microsoft.Azure.WebJobs.StorageProvider.Common? Trying to scope public surface changes to the new stopgap assemblies

Reply:
I think it can live in the StorageProvider.Common assembly. I originally put it in WebJobs.Host since it already had similar extension methods.

Reply:
Actually the GetWebJobsConnectionStringSection method is used in WebJobs.Script.WebHost so if we moved it into the StorageProvider.Common package, we would need to add that package directly I think. Since this is the only needed method, I can make the others private/internal, but this logic may be more suited to be in WebJobs.Host. Let me know what you think.